### DISCUSSION TITLE
DNSSEC results in query option only

### DISCUSSION THREAD

**Discussion by blixten85** [AdguardTeam/AdGuardHome#4513] [Ideas]
If you are running your own DNS server localy like Unbound. 
Since DNSSEC flag is already set, there should be an option in Adguard to just enable the DNSSEC results for the query log.
The DNSSEC option that is now sets both flag and checks the results. And one of them is unnecessary.

**Comment by jumpsmm7** [comment]
> If you are running your own DNS server localy like Unbound. Since DNSSEC flag is already set, there should be an option in Adguard to just enable the DNSSEC results for the query log. The DNSSEC option that is now sets both flag and checks the results. And one of them is unnecessary.

So what you mean is that you would like for users to be able to benefit from the dnssec of say for example, unbound, and see the flags in adguardhome log, but without having to redo the validation process once the query reaches adguardhome?

**Reply by blixten85** [reply]
Yes, that sounds exactly like what im looking for.
So a second option for DNSSEC for those who run unbound locally. 

I understand that those who does not run it locally might want to have the DNSSEC flag+result option left intact.
